### DISCUSSION TITLE
View All Songs

### DISCUSSION THREAD

**Discussion by spiredem** [SubmarinerApp/Submariner#190] [Q&A]
Is there any way I can view all songs?  I don’t want to go by artist or playlist, I want to see what I see in the playlist view but with all songs.

**Comment by NattyNarwhal** [comment]
Not yet, but as a workaround, if your server supports smart playlists, you can make a playlist that contains everything.

**Reply by spiredem** [reply]
You’re right. I did exactly that. Thank you!
